stdlib/2and3/mmap.pyi

```diff
@@ -71,6 +70,9 @@ if sys.version_info >= (3,):
         def __setitem__(self, index: int, object: int) -> None: ...
         @overload
         def __setitem__(self, index: slice, object: bytes) -> None: ...
+        # Doesn't actually exist, but the object is actually iterable because it has __getitem__ and
+        # __len__, so we claim that there is also an __iter__ to help type checkers.
```

Comment:
I don't have any code using mmap; I was just trying to fix an open typeshed issue. cc @euresti who opened the issue.

My thinking was that it's worse if people rely on mmap being iterable and get false positive type errors than if people somehow access mmap.__iter__ directly and don't get a type error.

I'm also not sure how protocols would solve this, since I don't think PEP 544 supports protocols like Iterable that can be fulfilled either through __iter__ or through __getitem__ and __len__.

I'm happy to change this though if you prefer dropping it.